llvm/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp

```diff
@@ -1121,7 +1132,7 @@ void ARMAsmBackend::applyFixup(const MCAssembler &Asm, const MCFixup &Fixup,
   assert(Offset + NumBytes <= Data.size() && "Invalid fixup offset!");
 
   // Used to point to big endian bytes.
-  unsigned FullSizeBytes;
+  unsigned FullSizeBytes = 0;
```

Comment:
Is this necessary for this patch? If not then it could be moved out into a separate NFC refactor.

Looking at the code FullSizeBytes is only used when big endian, so it is always initialized when used. Is this tripping over a static analysis tool?

Reply:
No, not necessary. I was modifying this code for something unrelated this snuck in, I'll just remove it.